pkgs/top-level/aliases.nix

```
@@ -185,6 +185,7 @@ mapAliases ({
   exfat-utils = exfat;                  # 2015-09-11
   facette = throw "facette has been removed."; # added 2020-01-06
   fast-neural-doodle = throw "fast-neural-doodle has been removed, as the upstream project has been abandoned"; # added 2020-03-28
+  fedora-coreos-config-transpiler = throw "fedora-coreos-config-transpiler has been renamed to 'butane'."; # added 2021-04-13
```

```suggestion
  fedora-coreos-config-transpiler = butane; # added 2021-04-13
```
This would auto upgrade the package if the users have aliases enabled.

---

I considered this, but the name of the binary has also changed from `fcct` to `butane`, so likely manual intervention is needed anyway. I think it’s better to fail explicitly here with a clear message, than to later run into a "no such file `fcct`" error somewhere. What do you think?